### PR TITLE
[Doc EN]: `guide/views.md` review 

### PR DESCRIPTION
**Pull request by Haeresis**
In our French translation/review we have changed some consistent things, so we suggest you do the same in the original documentation.

More details in comments.

Do not hesitate to set back the original behavior for some changes if you think they do not make sense.

### CODE REVIEW

File: en/guide/views.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
 title: Views
-description: The Views section describes all you need to configure data and views for a specific route in your Nuxt.js application. (Document, Layouts, Pages and HTML Head)
+description: The Views section describes all you need to configure data and views for a specific route in your Nuxt.js Application (Document, Layouts, Pages and HTML Head).
```
Comment (Haeresis) [inline en/guide/views.md]: To be more consistent with « Universal Nuxt.js Applications »

File: en/guide/views.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 ---
- 
-> The Views section describes all you need to configure data and views for a specific route in your Nuxt.js application. (Document, Layouts, Pages and HTML Head)
+
+> The Views section describes all you need to configure data and views for a specific route in your Nuxt.js Application (Document, Layouts, Pages and HTML Head).
```
Comment (Haeresis) [inline en/guide/views.md]: Same as above

File: en/guide/views.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 
 ![nuxt-views-schema](/nuxt-views-schema.png)
 
 ## Document
 
-> You can customise the main document with nuxt.js
+> You can customise the main document with Nuxt.js.
```
Comment (Haeresis) [inline en/guide/views.md]: Product name with capital. And final dot.

File: en/guide/views.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
 
-To extend the html template, create a `app.html` at the root of your project.
+To extend the HTML template, create a `app.html` at the root of your project.
```
Comment (Haeresis) [inline en/guide/views.md]: HTML
Source : https://www.w3.org/html/

File: en/guide/views.md [hunk 5]
```diff
@@ -110,7 +110,7 @@ export default {
 </script>
 ```
 
-More information about the layout property: [API Pages layout](/api/pages-layout)
+More information about the `layout` property: [API Pages `layout`](/api/pages-layout)
```
Comment (Haeresis) [inline en/guide/views.md]: tick because code part

File: en/guide/views.md [hunk 6]
```diff
@@ -130,7 +130,7 @@ export default {
     return { name: 'World' }
   },
   fetch () {
-    // The fetch method is used to fill the store before rendering the page
+    // The `fetch` method is used to fill the store before rendering the page
```
Comment (Haeresis) [inline en/guide/views.md]: tick because code part

File: en/guide/views.md [hunk 7]
```diff
@@ -150,14 +150,14 @@ export default {
 
 | Attribute | Description |
 |-----------|-------------|
-| asyncData | The most important key, it can be asynchronous and receives the context as argument, please read the [async data documentation](/guide/async-data) to learn how it works. |
-| fetch | Used to fill the store before rendering the page, it's like the data method except it doesn't set the component data. See the [API Pages fetch documentation](/api/pages-fetch). |
-| head | Set specific Meta Tags for the current page, see [API Pages head documentation](/api/pages-head). |
-| layout | Specify a layout defined in the `layouts` directory, see [API Pages layouts documentation](/api/pages-layout). |
-| transition | Set a specific transition for the page, see [API Pages transition](/api/pages-transition). |
+| asyncData | The most important key, it can be asynchronous and receives the context as argument, please read the [API Pages `asyncData` documentation](/guide/async-data) to learn how it works. |
```
Comment (Haeresis) [inline en/guide/views.md]: To be more consistent with others and some tick for code part

File: en/guide/views.md [hunk 8]
```diff
@@ -150,14 +150,14 @@ export default {
...
+| fetch | Used to fill the store before rendering the page, it's like the data method except it doesn't set the component data. See the [API Pages `fetch` documentation](/api/pages-fetch). |
+| head | Set specific Meta Tags for the current page, see [API Pages `head` documentation](/api/pages-head). |
+| layout | Specify a layout defined in the `layouts` directory, see [API Pages `layout` documentation](/api/pages-layout). |
+| transition | Set a specific transition for the page, see [API Pages `transition`](/api/pages-transition). |
 | scrollToTop | Boolean, by default: `false`. Specify if you want the page to scroll to the top before rendering the page, it's used for [nested routes](/guide/routing#nested-routes). |
-| validate | Validator function for a [dynamic route](/guide/routing#dynamic-routes). |
```
Comment (Haeresis) [inline en/guide/views.md]: To be consistent with nested routes.

File: en/guide/views.md [hunk 9]
```diff
@@ -194,10 +194,10 @@ head: {
 
 To know the list of options you can give to `head`, take a look at [vue-meta documentation](https://github.com/declandewet/vue-meta#recognized-metainfo-properties).
 
-More information about the head method: [API Configuration head](/api/configuration-head)
+More information about the `head` method: [API Configuration `head`](/api/configuration-head).
```
Comment (Haeresis) [inline en/guide/views.md]: Code part.

File: en/guide/views.md [hunk 10]
```diff
@@ -194,10 +194,10 @@ head: {
...
 
 ### Custom Meta Tags for a Page
 
-More information about the head method: [API Pages head](/api/pages-head)
+More information about the head method: [API Pages `head`](/api/pages-head).
```
Comment (Haeresis) [inline en/guide/views.md]: Code part.

File: en/guide/views.md [hunk 11]
```diff
@@ -53,6 +53,7 @@ You can extend the main layout by adding a `layouts/default.vue` file.
 *Make sure to add the `<nuxt/>` component when creating a layout to display the page component.*
 
 The default layout source code is:
+
```
Comment (Haeresis) [inline en/guide/views.md]: Space line convention in Markdown documentation

File: en/guide/views.md [hunk 12]
```diff
@@ -68,6 +69,7 @@ This layout is special since you should not include `<nuxt/>` inside its templat
 The default error page source code is [available on GitHub](https://github.com/nuxt/nuxt.js/blob/master/lib/app/components/nuxt-error.vue).
 
 Example of a custom error page in `layouts/error.vue`:
+
```
Comment (Haeresis) [inline en/guide/views.md]: Space line convention in Markdown documentation

File: en/guide/views.md [hunk 13]
```diff
@@ -102,6 +104,7 @@ Example of `layouts/blog.vue`:
 ```
 
 And then in `pages/posts.vue`, you can tell Nuxt.js to use your custom layout:
+
```
Comment (Haeresis) [inline en/guide/views.md]: Space line convention in Markdown documentation

**Comment by Haeresis** [comment]
I suggest you this third PR on five @Atinux

**Comment by Atinux** [comment]
:raised_hands:

**Pull request merged**